<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Techno-311" failures="1" tests="2" name="CancelAppointment" time="3.643" errors="0" timestamp="14 Jul 2023 16:29:50 GMT">
  <testcase classname="io.mosip.testscripts.PutWithPathParam" name="" time="1.819">
    <failure type="io.mosip.admin.fw.util.AdminTestException" message="Failed at output validation">
      <![CDATA[io.mosip.admin.fw.util.AdminTestException: Failed at output validation
	at io.mosip.testscripts.PutWithPathParam.test(PutWithPathParam.java:101)
	at io.mosip.testrunner.MosipTestRunner.startTestRunner(MosipTestRunner.java:161)
	at io.mosip.testrunner.MosipTestRunner.main(MosipTestRunner.java:114)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!--  -->
  <testcase classname="io.mosip.testscripts.PutWithPathParam" name="Prereg_CancelAppointment_All_Valid_Smoke_sid" time="1.819"/>
</testsuite> <!-- CancelAppointment -->
